python/mxnet/gluon/rnn/rnn_layer.py

```diff
@@ -286,6 +293,8 @@ class RNN(_RNNLayer):
         The recurrent state's shape is `(num_layers, batch_size, num_hidden)`.
         If `bidirectional` is True, state shape will instead be
         `(2*num_layers, batch_size, num_hidden)`
+        If input recurrent state is None, zeros are used as default begin states,
```

Comment:
Recurrent state shape -> Recurrent state.

and change the beginning sentence to The recurrent states is an NDArray with shape xxx.